src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas3/TransitionalAccommodationReferralReportsTest.kt

```diff
@@ -316,9 +317,12 @@ class TransitionalAccommodationReferralReportsTest : IntegrationTestBase() {
             withCreatedAt(OffsetDateTime.now())
             withDutyToReferLocalAuthorityAreaName("London")
             withDutyToReferSubmissionDate(LocalDate.now())
-            withHasHistoryOfArson(true)
+            withHasHistoryOfArson(false)
+            withIsConcerningArsonBehaviour(true)
             withIsDutyToReferSubmitted(true)
-            withIsRegisteredSexOffender(true)
+            withIsRegisteredSexOffender(null)
+            withIsHistoryOfSexualOffence(false)
```

Comment:
Just a minor comment, can this be `withHasHistoryOfSexualOffence` to make it grammatically correct and same as `HasHistoryOfArson`.